indexd/auth/drivers/alchemy.py

```diff
@@ -131,8 +131,8 @@ def authz(self, method, resource):
                 "Arborist is not configured; cannot perform authorization check"
             )
         if not resource:
-            # TODO: fix this. Setting authz to [] throws this error but
-            # admins should be able to do it
-            raise AuthError("Permission denied.")
+            # if the `authz` is empty, admins should still be able to perform
+            # operations on the record. For now, admin = access to `/programs`
+            resource = ["/programs"]
```

Comment:
`"*"` => arborist returns false - interprets it as a resource
`"/"` => arborist returns a "missing resource in auth request" error